### ISSUE TITLE
about the model output number

### ISSUE THREAD

**Issue by Usernamezhx** [bedapudi6788/NudeNet#48]
thanks for your work. it is very cool.  you said you collect two class data:safe and nude from the  [https://medium.com/@praneethbedapudi/nudenet-an-ensemble-of-neural-nets-for-nudity-detection-and-censoring-d9f3da721e3](url) . but the classification data you provide is three class:nude safe sexy. so the output number is 2? you combine the sexy and nude to one class? is that right?

**Comment by bedapudi6788** [comment]
Yes, that is correct. sexy + nude is unsafe and safe is safe.

**Comment by Usernamezhx** [comment]
thanks for your reply. i train model with three class. use the seresnet50 as the backbone and finetune all layer with the imagenet model file. the loss is a little strange. Can you provide the train log about the acc and loss if you want.
this is my train log. I set batch size to 128.
```python
LR:0.001 Training: Loss: 0.2197, Accuracy: 0.9197, Validation : Loss : 0.4271,  Accuracy: 0.8646, Time: 2284.8669s
LR:0.001 Training: Loss: 0.1424, Accuracy: 0.9486, Validation : Loss : 0.3773,  Accuracy: 0.8840, Time: 2300.8871s
LR:0.001 Training: Loss: 0.1224, Accuracy: 0.9558, Validation : Loss : 0.3751,  Accuracy: 0.8866, Time: 2290.4965s
LR:0.001 Training: Loss: 0.1100, Accuracy: 0.9602, Validation : Loss : 0.4759,  Accuracy: 0.8667, Time: 2310.8288s
```

**Comment by bedapudi6788** [comment]
> Can you provide the train log about the acc

I don't have the train log saved for the classifier. Sorry!

I haven't used seresnet before. Not sure how it is supposed to progress.

**Comment by Usernamezhx** [comment]
thanks for your reply. give time a little time. the loss maybe go down :p
